.github/ISSUE_TEMPLATE/-maintainer--new-feature.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "(Maintainer) New Feature"
+about: New feature description, filed by CRM maintainers
+title: ''
+labels: enhancement, source::maintainer
+assignees: ''
+
+---
+
+**What is the idea?**
+Describe what the feature is and the desired state.
```

Comment:
Should these be commented-out? Does this explanation get auto-filled into the issue? Like am I going to have to delete line 11 every time I fill this out?